2020-04-introducing-mdjs-live-demos-everywhere/README.md

```diff
@@ -0,0 +1,360 @@
+Install a chrome extension and suddenly github gets superpowers.
+
+All that is needed is to have Markdown files with the correct code fence blocks.
+And yeah your code needs to be able to run from within [unpkg.com](https://unpkg.com).
+
+**How does it actually work?**
+
+The extension detects on which github page you are.
+If it actually finds a markdown file or an issue with mdjs code then it adds a "show demo" button to can active it.
+Only if you click the button it will start gathering all the needed info.
+
+- find the nearest `package.json`
+- read the actual markdown file/issue content
+- replace all bare import with `unpkg.com` imports
+- replace all relative imports with `unpkg.com` and the name of the package.json + relative path for it
+- create a secured iframe
+- position the iframe absolute as an overlays
+- put the jsCode and html code inside the iframe
+- the button becomes a toggle to show/hide the iframe
+
+Some of the tasks are more complicated and require some extra work to make it secure but in the sense that's it.
+
+With that, you can put documentation with live examples on github.
+Even issues with demos showing the actual error in the code are possible.
+
+That sure sounds like a hell of a tool to improve your documentation an issue reproduction.
+Especially as the readme & issue content still remain useful even without the extension.
+
+For more detailed information please see [https://github.com/open-wc/mdjs-viewer](https://github.com/open-wc/mdjs-viewer).
+
+### Supported on webcomponents.dev
+
+Fully supported by this awesome online editor.
+
+![webcomponent.dev screenshot](https://raw.githubusercontent.com/open-wc/blog-posts/feat/mdjs/2020-04-introducing-mdjs-live-demos-everywhere/images/webcomponents-dev-screenshot.png)
+
+You can directly edit your documentation, demos and code directly in the browser.
+
+![webcomponent.dev screenshot](https://raw.githubusercontent.com/open-wc/blog-posts/feat/mdjs/2020-04-introducing-mdjs-live-demos-everywhere/images/webcomponents-dev-new.png)
+
+You can start directly with documentation as in the screenshot above, or even better you can use it in every Markdown file or README.md 💪
+
+Give it a go and document your components in all it's glory.
+
+All the demo links are actually from [webcomponents.dev](https://webcomponents.dev/edit/collection/lsZ2eaviDNwy6pIBEDeL/tS7JYfymt6yeshma8Gn1?pm=1&sv=1).
+
+Be sure to [check it out](https://webcomponents.dev/).
+
+## How you can add support for mdjs
+
+Please check the official documentation page at [https://open-wc.org/mdjs/](https://open-wc.org/mdjs/).
+
+## Resume
+
+There you have it - mdjs is a format that can be shown in many different ways.
+It is your single source of truth for good looking documentation everywhere.
+Be it your locally, a published storybook, on github or npmjs it always looks good even if there is no direct support for it. If possible it will become interactive demos through progressive enhancement.
```

Comment:
```suggestion
Be it locally, a published storybook, on github or npmjs it always looks good even if there is no direct support for it, but when possible it will become interactive demos through progressive enhancement.
```